cmd/cni-metrics-helper/metrics/metrics_test.go

```diff
@@ -17,16 +17,20 @@ import (
 	"io/ioutil"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/publisher"
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
+	"github.com/stretchr/testify/assert"
 )
 
 type testMetricsTarget struct {
 	metricFile         string
 	interestingMetrics map[string]metricsConvert
 }
 
+func (target *testMetricsTarget) getLogger() logger.Logger {
+	return logger.DefaultLogger()
+}
```

Comment:
you wouldn't need this interface method if you just made all those functions instead be methods on metricsTarget.